File: Selection-Sort_ Decrease.py
class Solution:
    def selectionSort(self, nums):
        for i in range(len(nums)):
            minV =i 
            for j in range(len(nums)-1):
                if nums[i]>nums[j]:
                    nums[minV],nums[j] = nums[j],nums[minV]
        return nums
n=input().strip("[]")
n=list(map(int,n.split(",")))
print(Solution().selectionSort(n))

#Input: nums = [7, 4, 1, 5, 3]
#Output: [7, 5, 4, 3, 1]

File: Selection-Sort-decrease.js
class Solution {
    selectionSort(nums) {
        for (let i = 0; i < nums.length; i++) {
            let minIndex = i;
            for (let j = i + 1; j < nums.length; j++) {
                if (nums[j] > nums[minIndex]) {
                    minIndex = j;
                }
            }
            [nums[i], nums[minIndex]] = [nums[minIndex], nums[i]];
        }
        return nums;
    }
}
const u = [7, 4, 1, 5, 3];
console.log(new Solution().selectionSort(u));
